pkgs/development/python-modules/python-lsp-server/default.nix

```diff
@@ -142,6 +142,7 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [
     "pylsp"
+    "pylsp.python_lsp"
```

Comment:
This is what __main__ imports but it doesn't reproduce the error, probably nativeBuildInputs is still available during check phase